paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -37,17 +39,16 @@ class PaparazziPlugin : Plugin<Project> {
     val variants = project.extensions.getByType(LibraryExtension::class.java)
         .libraryVariants
     variants.all { variant ->
+      val paparazziResourcesDetailsFile = "intermediates/paparazzi/${variant.name}/resources.txt"
       val variantSlug = variant.name.capitalize(Locale.US)
-
       val writeResourcesTask = project.tasks.register(
           "preparePaparazzi${variantSlug}Resources", PrepareResourcesTask::class.java
       ) {
-        // TODO: variant-aware file path
-        it.outputs.file("${project.buildDir}/intermediates/paparazzi/resources.txt")
+        it.outputs.file("${project.buildDir}/${paparazziResourcesDetailsFile}")
```

Comment:
is it possible to set this using the same layout.buildDirectory style below?

Reply:
I wonder if marking `outputResourcesFile` as `@OutputFile` is enough in this case. I'll do that.